Generate_graph.py

```diff
@@ -15,6 +15,9 @@
 OUTPUT_FILE = '{}_graph.json'
 CONFIG_FILE = 'config.yml'
 
+MOBILITY_DIR = 'MobilityMatrix'
+MOBILITY_FILENAME = '{country_iso3}_mobility_matrix.csv'
```

Comment:
I'm not sure if it might be better to create a totally separate mobility directory, which the user could sym link to the output directory of the mobility matrix repo.

Alternatively, mobility_directory could be an input argument to this script, and if not provided would assume the default case that both repos are in the same directory and have their original names

Reply:
Options:
1. The scripts download the mobility csv files from the Github repo just like they download the other files
2. Symbolic link to the other repo
3. Parameter to pass in the path

For the most part, these scripts are more convention over configuration in approach so I would typically choose 1 in that case. I'm fine with 3. Would option 2 work for people using WIndows? I haven't used it in a long time but I don't remember it supporting symlinks.

Reply:
Oh right, good point about number 2. I hadn't thought of 1, that's a good idea, although I think the mobility matrix data is perhaps more likely to change during the duration of this project than our other data sources. So perhaps I would vote for 3, what do you think?

Reply:
I'll implement option 3 with the command line argument -m ( or --mobility).

Reply:
@turnerm I added the command line parameter, added some documentation to the readme, and set the country code on the graph in a better location